modules/serverless-openshift-routes.adoc

```diff
@@ -30,12 +35,13 @@ kind: Service
 metadata:
   name: <service_name>
   annotations:
-    serving.knative.openshift.io/disableRoute: true
+    serving.knative.openshift.io/disableRoute: "true"
 spec:
   template:
     spec:
       containers:
       - image: <image>
+...
```

Comment:
This is apparently invalid YAML. Jason has a solution out for review here: https://github.com/openshift/openshift-docs/pull/35814/files#diff-719cf3b36e5cfdc45393ebde8ff09fd9a56c239b5671fe1cea5b7f364d647125R914

Reply:
ACK - I think for now I'll leave it as is so that our docs are consistent, but if Jason's solution is implemented it'd make sense to go through our Serverless docs and update all instances of this together.

FWIW I'm not sure if it matters that the YAML is invalid, since the point of this is to show that it's an excerpt, so if this was all you copied into a file, it would be invalid anyway?

Reply:
Because if someone copies the entire block and uses it as basis for their YAML file, they'd have to prune the invalid line, too. (And it scans like that YAML is supposed to be the entire definition waiting to be customized.)

Reply:
+1